pkg/bindings/generate/types_systemd_options.go

```diff
@@ -11,8 +11,6 @@ import (
 
 /*
 This file is generated automatically by go generate.  Do not edit.
-
-Created 2020-12-18 15:58:20.661450253 -0600 CST m=+0.000135779
```

Comment:
why did you remove this?

Reply:
Because this will cause us to have to update the bindings every time you run create bindings even if the bindings did not change.